examples/portfolio.py

```diff
@@ -62,6 +62,10 @@ def f(x):
     s.add('x', weights_vector, 3 * 3)
 
     qubo, offset = SamplingCompiler.generate_qubo_matrix(f, s.size, s)
+    if SamplingCompiler.test_qubo_matrix(f, qubo, offset):
+        print("QUBO generation successful")
```

Comment:
I think we need to wait with portfolio for testing that is using custom search space mapping, as input to f is not just the binary vector, and needs to be mapped to a uint vector.